PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
+
+    #URL = "https://api.develop.socotra.com/account/authenticate"
```

Comment:
While there are different schools of thought on how comments can and should be used, you should never need to comment out code.